akka-docs-dev/rst/java/migration-guide-1.0-2.x-java.rst

```diff
@@ -357,4 +357,44 @@ We show the necessary steps in terms of an example ``AsyncStage``
 Example
 ^^^^^^^
 
-TODO
\ No newline at end of file
+TODO
+
+Akka HTTP: Uri parsing mode relaxed-with-raw-query replaced with rawQueryString
+===============================================================================
+
+Previously Akka HTTP allowed to configure the parsing mode of an Uri's Query part (``?a=b&c=d``) to ``relaxed-with-raw-query``
+which is useful when Uris contain not the usual syntax (not key/value pairs).
```

Comment:
made it: `which is useful when Uris are not formatted using the usual "key/value pairs" syntax.`